linux_pipeline/Jenkinsfile_kernel_bisect_commit_validator

```diff
@@ -1,193 +1,267 @@
-        overrideIndexTriggers(false)
+def PowerShellWrapper(psCmd) {
+    psCmd = psCmd.replaceAll("\r", "").replaceAll("\n", "")
+    bat "powershell.exe -NonInteractive -ExecutionPolicy Bypass -Command \"\$ErrorActionPreference='Stop';[Console]::OutputEncoding=[System.Text.Encoding]::UTF8;$psCmd;EXIT \$global:LastExitCode\""
+}
+
+properties ([
+    [$class: 'ParametersDefinitionProperty',
+        parameterDefinitions: [
+        [$class: 'ChoiceParameterDefinition',
+            choices: 'git://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git\nhttps://git.kernel.org/pub/scm/linux/kernel/git/davem/net-next.git\nazure_kernel',
+            name: 'KERNEL_GIT_URL',
+            description: 'Job to run bisect on'],
+        [$class: 'StringParameterDefinition',
+            name: 'KERNEL_GIT_BRANCH',
+            defaultValue: 'master',
+            description: 'Branch to be built'],
+        [$class: 'StringParameterDefinition',
+            name: 'KERNEL_GIT_COMMIT_ID',
+            defaultValue: 'HEAD',
+            description: 'Specifiv commit id to build'],
+        [$class: 'ChoiceParameterDefinition',
+            choices: 'Ubuntu\nCentOS',
+            name: 'DISTRO_VERSION',
+            description: 'Distro version'],
+        [$class: 'StringParameterDefinition',
+            name: 'ENABLED_STAGES',
+            defaultValue: 'build_artifacts, boot_test',
+            description: 'What stages to run. By default all stages are enabled'],
+        [$class: 'ChoiceParameterDefinition',
+            choices: 'TCP\nUDP\nFIO',
+            name: 'PERF_TEST',
+            description: 'Which perf test to run'],
+        [$class: 'ChoiceParameterDefinition',
+            choices: 'v4\nv6',
+            name: 'PERF_IPV',
+            description: 'IP version to use.(Only used for networking perf test)'],
+        [$class: 'ChoiceParameterDefinition',
+            choices: '1k\n8k',
+            name: 'PERF_K',
+            description: 'K to be used, UDP only'],
+        [$class: 'ChoiceParameterDefinition',
+            choices: '1\n2\n4\n8\n16\n32\n64\n128\n256\n512\n1024\n2048\n4096\n6144\n8192\n10240',
+            name: 'PERF_TEST_THREADS_TCP',
+            description: 'TEST_THREADS for TCP'],
+        [$class: 'ChoiceParameterDefinition',
+            choices: '1\n2\n4\n8\n16\n32\n64\n128\n256\n512\n1024',
+            name: 'PERF_TEST_THREADS_UDP',
+            description: 'TEST_THREADS for UDP'],
+        [$class: 'ChoiceParameterDefinition',
+            choices: '1\n1024',
+            name: 'PERF_IO_SIZE',
+            description: 'IO_SIZE for FIO'],
+        [$class: 'ChoiceParameterDefinition',
+            choices: '1\n2\n4\n8\n16\n32\n64\n128\n256',
+            name: 'PERF_Q_DEPTH',
+            description: 'Q_DEPTH for FIO'],
+        [$class: 'ChoiceParameterDefinition',
+            choices: 'read\nrandread\nwrite\nrandwrite',
+            name: 'PERF_IO_MODE',
+            description: 'IO_MODE for FIO']
```

Comment:
ditto -- make all these description human readable 
@chvalean  please advise on how the descriptions should look like